### DISCUSSION TITLE
On my Surface device, confetti shoots radially down. What's causing this?

### DISCUSSION THREAD

**Discussion by viktorb1** [matteobruni/tsparticles#5134] [General]
https://github.com/matteobruni/tsparticles/assets/14933511/3df875f5-dc6c-45cf-80bf-2e85446a838f

I used the site https://confetti.js.org/more.html to test the confetti and for some reason it shoots in a weird away. On most of my computers the confetti shoots up and then falls down, but on this computer it shoots radially down. Any ideas why that might be the case? Seems to be happening on all my browsers on that computer.

**Comment by matteobruni** [comment]
Sorry but I can't reproduce the issue, it works fine on all my devices. Let me know if you find something that can be helpful reproducing the issue in other devices too.

**Comment by viktorb1** [comment]
Yea, me neither. I have two other Surface devices and they don't have this problem. My initial gut says it might be a touchscreen driver related issue, but I have no idea because my touchscreen works just fine. I tried disabling it and the issue still persists. I also tried it in Safe Mode and it happens there as well, in all browsers.

For context, I am testing on:
Microsoft Surface Pro 6 (Intel Core i5, 8GB RAM, 256GB)
Windows 11 - Version 10.0.22621 Build 22621
All latest versions of browsers

Of course, I don't expect this to get fixed because I cannot reproduce it on any other device. Just wanted to let you know it's happening...
